main-cookbooks/teracy-dev/recipes/rbenv.rb

```diff
@@ -32,48 +32,51 @@
 #
 
 if node['teracy-dev']['ruby']['enabled']
-
-    node.default['rbenv']['user']           = 'vagrant'
-    node.default['rbenv']['group']          = 'vagrant'
-    node.default['rbenv']['user_home']      = '/home/vagrant'
-
     include_recipe 'rbenv::default'
     include_recipe 'rbenv::ruby_build'
 
-    node_version = node['teracy-dev']['ruby']['version']
 
-    if node_version.strip().empty?
-        begin
-            versions = []
-            list_versions = Mixlib::ShellOut.new('rbenv install -l').run_command.stdout
+    node['teracy-dev']['ruby']['versions'].each do |ruby_version|
+        if ruby_version.strip().empty?
+            begin
+                versions = []
+                list_versions = Mixlib::ShellOut.new('rbenv install -l').run_command.stdout
```

Comment:
the logic here should be updated, `rbenv` is supposed not to be available on the system (and it is), so don't process the empty value, just ignore the empty values.